theatre_cli/config.py

```diff
@@ -1,6 +1,8 @@
 import json
-
-CONFIG_FILE = "config.json"
+import os
+#Changing the target path to the new folder that we created accodring to linux folder structure conventions
```

Comment:
@lexcsam Can you fix the spelling mistake OR changing the sentence and give space after the `#`?
```suggestion
# Using default .config directory for storing configuration file
```